feat: Add new Shipwright models

jerolimov opened the pull request:
Hi @tommy351, I really appreciate your work here.👍

This PR adds CRD-generated models for [Shipwright](https://shipwright.io/). See the committed README.md and CHANGELOG.md. Shipwright is a framework for building container images.

~~I would be happy if you could release this version as 0.9.0. After that, I would create updates for 0.10.0 and 0.11.0.~~

~~Also if the changes between these versions are super small and add only additional fields, I would like to go with a 0.9 release first because I know that this version is still used in many environments.~~

~~Let me know if this is fine for you.~~

edit: It uses the latest Shipwright 0.11.0 CRDs now. See my comment below.

tommy351 commented:
Are there any CRD breaking changes between 0.9.0, 0.10.0 and 0.11.0? If not, I think it would be fine just using the latest CRD files.

jerolimov commented:
Hi @tommy351, I double-checked the difference and can confirm that the newer versions only add additional properties to the types.

In theory, this means that a 0.11 generated type would allow "too much" on a 0.9 cluster. But I agree that this can happen with most resources. And since the API version is still **v1alpha1** this could happen "at any time".

The package.json now uses the latest 0.11.0 version. I've also applied similar changes from #139: Removed the CHANGELOG, updated version in package.json to 0.0.0 and switched the changeset level to `minor`.

Please let me know if anything else is needed.
